packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts

```diff
@@ -34,6 +34,9 @@ class CardPayWithdrawalWorkflowTransactionStatusComponent extends Component<Work
       this.completedCount = 2;
       this.args.onComplete?.();
     } catch (e) {
+      console.error('Failed to complete bridging to layer 1');
+      console.error(e);
+      Sentry.captureException(e);
```

Comment:
Does this look correct to you, @lukemelia? My understanding is Sentry doesn't automatically capture browser logs and requires manual capturing of caught exceptions.

Reply:
Based on: https://docs.sentry.io/platforms/javascript/guides/ember/usage/

Reply:
I believe that console messages are captured as breadcrumbs but would not be treated as an error, so the approach you have here is good.